lib/Dialect/FIRRTL/Transforms/Dedup.cpp

```diff
@@ -54,9 +54,8 @@ llvm::raw_ostream &printHex(llvm::raw_ostream &stream,
 }
 
 llvm::raw_ostream &printHash(llvm::raw_ostream &stream, llvm::SHA256 &data) {
-  auto string = data.result();
-  ArrayRef bytes(reinterpret_cast<const uint8_t *>(string.begin()),
-                 string.size());
+  std::array<uint8_t, 32> hash = data.result();
```

Comment:
can this just be return printHex(stream, data.result()) ?

Reply:
Yep, looks like that works, thanks! Just pushed up an amended commit with that suggestion.